Fix deprecation messages in `AdminResettingController` and `AdminResettingController`

phansys opened the pull request:
<!-- THE PR TEMPLATE IS NOT AN OPTION. DO NOT DELETE IT, MAKE SURE YOU READ AND EDIT IT! -->
## Subject
Fix deprecation messages in `AdminResettingController` and `AdminResettingController`, since currently they are making references to non existing classes.
<!-- Describe your Pull Request content here -->

<!--
    Show us you choose the right branch.
    Different branches are used for different things :
    - 4.x is for everything backwards compatible, like patches, features and deprecation notices
    - master is for deprecation removals and other changes that cannot be done without a BC-break
    More details here: https://github.com/sonata-project/SonataUserBundle/blob/4.x/CONTRIBUTING.md#the-base-branch
-->
I am targeting this branch, because these changes respect BC.

<!--
    Specify which issues will be fixed/closed.
    Remove it if this is not related.
-->

<!--
    If this is a work in progress, uncomment the "To do" section.
    You can add as many tasks as you want.
    If some are not relevant, just remove them.
-->
<!--
## To do

- [ ] Update the tests;
- [ ] Update the documentation;
- [ ] Add an upgrade note.
-->

jordisala1991 commented:
There might me something wrong on tests:

```
Since symfony/phpunit-bridge 5.1: Using "@expectedDeprecation" annotations in tests is deprecated, use the "ExpectDeprecationTrait::expectDeprecation()" method instead.
```

seems like we using the old way of expecting deprecations. Deprecated 😅

phansys commented:
>There might me something wrong on tests:

I've replaced those annotations. The test should be fine now.

jordisala1991 commented on tests/Controller/AdminResettingControllerTest.php:
```diff
@@ -20,12 +20,15 @@
 
 class AdminResettingControllerTest extends TestCase
 {
+    use ExpectDeprecationTrait;
```
I think this is only needed if you using really old versions of phpunit. on 8.5 we should be fine without it

phansys replied:
I can confirm it is required. We are using phpunit/phpunit:8.5.0:
https://github.com/sonata-project/SonataUserBundle/blob/d60215ea263d81ee2e40ff9dc2df795c6ca1d299/phpunit.xml.dist#L35

Please, see symfony/symfony#37153.

jordisala1991 replied:
Oh I see, needed then, I was confused with the expectException

VincentLanglet commented:
There should be no changelog, for pedantic isn't it ?

And I think two commit would be better.
- One with the deprecation message fix
- One removing the deprecated `@expectedDeprecation`. (Not sure if it was really needed btw, since we'll remove all of them in the next major ; all the other SonataBundle use this annotation, so now we're in a not-consistent situation 😅)

phansys commented:
Changelog section removed, since these changes are pedantic. It was:
## Changelog

<!-- MANDATORY
    Fill the changelog part inside the code block.
    Follow this schema: http://keepachangelog.com/
    This will end up on https://github.com/sonata-project/SonataUserBundle/releases,
    please keep it short and clear and to the point
-->

<!--
    If you are updating something that doesn't require
    a release, you can delete the whole "Changelog" section.
    (eg. update to docs, tests)
-->

<!-- REMOVE EMPTY SECTIONS -->
```markdown
### Fixed
- Fixed QCNs referenced by deprecation messages at `AdminResettingController` and `AdminResettingController`;
- Fixed deprecated usage of `@exptectedDeprecation` through `ExpectDeprecationTrait::expectDeprecation()`.
```

phansys commented:
>There should be no changelog, for pedantic isn't it ?

Changelog removed.

>And I think two commit would be better.

Commits splitted.

jordisala1991 commented:
Thank you @phansys
